fdbserver/Resolver.actor.cpp

```diff
@@ -501,6 +501,8 @@ ACTOR Future<Void> resolveBatch(Reference<Resolver> self,
 		}
 
 		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
+			// Unicast requires resolver private mutations to be enabled.
+			ASSERT(useResolverPrivateMutations);
```

Comment:
Place this in the "else" clause below and remove the "if (useResolverPrivateMutations)" condition there?